update-models.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+export BASE_DIR=$(pwd);
```

Comment:
I believe this will be the directory from where the user runs the script, not necassarily the directory of the script itself. Try `BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`